.github/workflows/bump_submodules.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Submodule
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
+  pull_request:
+    branches: ["main"]
```

Comment:
This syntax can be simplified (remove `["` and `"]`). Examples in-tree:
* https://github.com/iree-org/iree/blob/c2571c871c0254e993a01cee92ec9152823a9db2/.github/workflows/ci.yml#L31-L34
* https://github.com/iree-org/iree/blob/c2571c871c0254e993a01cee92ec9152823a9db2/.github/workflows/pkgci.yml#L11-L18